sweepbatcher/sweep_batch.go

```diff
@@ -495,6 +499,16 @@ func (b *batch) checkSweepToAdd(_ context.Context, sweep *sweep) (bool, error) {
 		return false, fmt.Errorf("the sweep is nil")
 	}
 
+	// If the sweep has a change output it must carry more than dust value.
+	if sweep.change != nil &&
+		sweep.value < dustLimitForPkScript(sweep.change.PkScript) {
+
+		return false, fmt.Errorf("sweep %x has a change output "+
+			"with value %d below dust limit %d",
+			sweep.swapHash[:6], sweep.change.Value,
+			dustLimitForPkScript(sweep.change.PkScript))
+	}
+
```

Comment:
Actually, this seems to be unneeded. We have this check in `constructUnsignedTx` which is called from `presign` which is called by `addSweeps`. So if the change is dust, it won't enter anyway.

I propose to remove this check and to add a test similar to `testPresigned_presigned_group_with_change` testing that trying to add too large or too small change (making one of the outputs dust) fails in both `PresignSweepsGroup` and subsequent `AddSweep`. Maybe we can even check it in the existing test as failed attempts before the succeeded attempt. Or just duplicate the test.

Reply:
added a separate test `testPresigned_presigned_group_with_dust_change`.